### PR TITLE
Chore/fix rubocop

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #14

### CODE REVIEW

File: lib/suma/processor.rb [hunk 1]
```diff
@@ -12,10 +12,25 @@
 module Suma
   class Processor
     class << self
-      def run(metanorma_yaml_path:, schemas_all_path:, compile:,
-output_directory: "_site")
-        Utils.log "Current directory: #{Dir.getwd}"
```
Comment (opoudjis) [inline lib/suma/processor.rb:17]: You got rid of this line, but TBH, it won't be missed

File: lib/suma/processor.rb [hunk 2]
```diff
@@ -28,53 +43,49 @@ def run(metanorma_yaml_path:, schemas_all_path:, compile:,
...
+          # File.delete(file)
         end
+      end
 
+      def build_collection(collection_config, output_directory)
         new_collection_config_path = "collection-output.yaml"
         collection_config.manifest.remove_schemas_only_sources
         collection_config.to_file(new_collection_config_path)
 
-        if compile
```
Comment (opoudjis) [inline lib/suma/processor.rb:55]: This is simplfying the flow a little too much.  I am happy to see spaghetti unravelled, but this was not my workflow, it was @koonwa and @ronaldtse , and I prefer them to weigh in on whether they need this code:

````
col = Suma::SchemaCollection.new(
           config_yaml: schemas_all_path,
           manifest: collection_config.manifest,
           output_path_docs: "schema_docs",
           output_path_schemas: "plain_schemas",
         )
````

and 

````
        new_collection_config_path = "collection-output.yaml"
         collection_config.manifest.remove_schemas_only_sources
         collection_config.to_file(new_collection_config_path)
````

to be run regardless of whether the `compile` variable is set, as is the case in the current code.

**Review by opoudjis — APPROVED** [review]
I approve, but I want @ronaldtse or @koonwa to weigh in on the flow simplifications in lib/suma/processor.rb

File: lib/suma/processor.rb [hunk 3]
```diff
@@ -12,10 +12,25 @@
 module Suma
   class Processor
     class << self
-      def run(metanorma_yaml_path:, schemas_all_path:, compile:,
-output_directory: "_site")
-        Utils.log "Current directory: #{Dir.getwd}"
+      def run(metanorma_yaml_path:, schemas_all_path:, compile:, output_directory: "_site")
+        Utils.log "Writing #{schemas_all_path}..."
```
Comment (CAMOBAP) [inline lib/suma/processor.rb:16]:
```suggestion
        Utils.log "Current directory: #{Dir.getwd}, writing #{schemas_all_path}..."
```

File: lib/suma/processor.rb (hunk 1 above)
Comment (CAMOBAP) [reply lib/suma/processor.rb:17]: merged it with "Writting ..." log

File: lib/suma/processor.rb (hunk 2 above)
Comment (CAMOBAP) [reply lib/suma/processor.rb:55]: Agreed, I really need a review from you @koonwa  and @ronaldtse

Comment (CAMOBAP) [reply lib/suma/processor.rb:55]: @koonwa @ronaldtse any feedback appreciated

**Pull request merged**